src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -61,49 +73,41 @@ public Shooter() {
   public void configure() {
     // -- Left Motor --
     leftConfig.MotorOutput.Inverted = constShooter.LEFT_INVERT;
-    leftConfig.Slot0.kV = prefShooter.leftShooterV;
-    leftConfig.Slot0.kS = prefShooter.leftShooterS;
-    leftConfig.Slot0.kA = prefShooter.leftShooterA;
-    leftConfig.Slot0.kP = prefShooter.leftShooterP;
-    leftConfig.Slot0.kI = prefShooter.leftShooterI;
-    leftConfig.Slot0.kD = prefShooter.leftShooterD;
-
-    leftConfig.MotionMagic.MotionMagicAcceleration = 400;
-    leftConfig.MotionMagic.MotionMagicJerk = 4000;
+    leftConfig.Slot0 = constShooter.LEFT_PID_SLOT_0_FAST;
+    leftConfig.Slot1 = constShooter.LEFT_PID_SLOT_1_SLOW;
+
+    leftConfig.MotionMagic.MotionMagicCruiseVelocity = 60;
+    leftConfig.MotionMagic.MotionMagicAcceleration = 600;
+    leftConfig.MotionMagic.MotionMagicJerk = 6000;
     leftMotor.getConfigurator().apply(leftConfig);
 
     // -- Right Motor --
     rightConfig.MotorOutput.Inverted = constShooter.RIGHT_INVERT;
-    rightConfig.Slot0.kV = prefShooter.rightShooterV;
-    rightConfig.Slot0.kS = prefShooter.rightShooterS;
-    rightConfig.Slot0.kA = prefShooter.rightShooterA;
-    rightConfig.Slot0.kP = prefShooter.rightShooterP;
-    rightConfig.Slot0.kI = prefShooter.rightShooterI;
-    rightConfig.Slot0.kD = prefShooter.rightShooterD;
-
-    rightConfig.MotionMagic.MotionMagicAcceleration = 400;
-    rightConfig.MotionMagic.MotionMagicJerk = 4000;
+    rightConfig.Slot0 = constShooter.RIGHT_PID_SLOT_0_FAST;
+    rightConfig.Slot1 = constShooter.RIGHT_PID_SLOT_1_SLOW;
+
+    rightConfig.MotionMagic.MotionMagicCruiseVelocity = 60;
+    rightConfig.MotionMagic.MotionMagicAcceleration = 600;
+    rightConfig.MotionMagic.MotionMagicJerk = 6000;
     rightMotor.getConfigurator().apply(rightConfig);
 
     // -- Pivot Motor --
     pivotConfig.Feedback.SensorToMechanismRatio = constShooter.PIVOT_GEAR_RATIO;
     pivotConfig.MotorOutput.Inverted = constShooter.PIVOT_INVERT;
     pivotConfig.MotorOutput.NeutralMode = constShooter.PIVOT_NEUTRAL_MODE;
-    pivotConfig.Slot0.kS = prefShooter.pivotShooterS;
-    pivotConfig.Slot0.kV = prefShooter.pivotShooterV;
-    pivotConfig.Slot0.kG = prefShooter.pivotShooterG;
-    pivotConfig.Slot0.kA = prefShooter.pivotShooterA;
-    pivotConfig.Slot0.kP = prefShooter.pivotShooterP;
-    pivotConfig.Slot0.kI = prefShooter.pivotShooterI;
-    pivotConfig.Slot0.kD = prefShooter.pivotShooterD;
     pivotConfig.Slot0.GravityType = constShooter.PIVOT_GRAVITY_TYPE;
```

Comment:
This should move int slot0 config. I think we are overriding it currently as setting slot0 occurs below